administrator/components/com_content/models/article.php

```diff
@@ -324,6 +324,7 @@ public function getItem($pk = null)
 		if (!empty($menuItems))
 		{
 			$item->menulink = array(
+			    'menuid' => $menuItems[0]->id,
```

Comment:
Yeah...I noticed it... And changed it just after the commit.. This will get corrected in next commit... :)